ark/settings.py

```diff
@@ -1,17 +1,17 @@
 # hope u liek capital letters
 
 # default segmented csv column names
-CELL_SIZE = 'cell_size'                     # cell size
-CELL_LABEL = 'label'                        # cell label number (regionprops)
-FOV_ID = 'SampleID'                         # cell's fov name
-CELL_TYPE = 'cell_type'                     # cell type name (flowsom)
-CLUSTER_ID = 'FlowSOM_ID'                   # cell cluster id (flowsom)
-PATIENT_ID = 'PatientID'                    # cell's patient id
-KMEANS_CLUSTER = 'cluster_labels'           # generated cluster column name
+CELL_SIZE = 'cell_size'  # cell size
```

Comment:
my tabs! 😢

Reply:
Oh sorry, I think that was Pycharm's fault!